### ISSUE TITLE
retrying network roles does not 're-up' the interface

### ISSUE THREAD

**Issue by dudymas** [opencrowbar/core#451]
if I have a node with a network role, and that role has a higher priority router.

When I retry the network role

I expect the node to have a new default route (to the higher priority router/gateway)

Instead the network settings are only set.
Going to the node and calling 'ifdown / ifup' makes the new default gateway takeover.

----

I expect for the best fix to be to get the interface for a given network and run a 'ifdown / ifup' on the associated interface via ipmi. That should do the trick, AFAIK.

**Comment by sjensen** [comment]
Might want to be careful w an ifdown/ifup.  I find that can cause probs like cutting your access off at the knees. Perhaps the interactive network cmds like route add and ifconfig?   Depending on the OS. However I thought CB already did this in most cases.

**Comment by galthaus** [comment]
Yeah. The more I think about the specifics.  I bet this is a router case bug in the chef network recipe.  Router was there and may need some review to make router changes are applied when addressing doesn't change

**Comment by kathpanderson** [comment]
This issue has been removed from "in progress"... It could likely be closed as well.
